src/Components/FormCreation/Dialogs/ManageApplicantAccessDialog.js

```diff
@@ -0,0 +1,294 @@
+import React, { useEffect } from "react";
```

Comment:
This file is not done. It is out of scope of the tickets and will be implemented with https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=0c79c8d83afd4d18bc18174c66c47d24